rc.d/dsbdriverd.tmpl

```diff
@@ -12,17 +12,28 @@
 #				Set it to YES to enable dsbdriverd.
 # dsbdriverd_flags (str):	Flags passed to dsbdriverd on startup.
 #				Default is "".
-#
+# dsbdriverd_blacklist (str):	Kernel excluded from loading space separated,
+#                               without .ko extension).
+#				Default is "".
 
 . /etc/rc.subr
 
 name=dsbdriverd
-command=@PATH_PROGRAM@
+desc="automatically try to load the suitable driver for your PCI and USB hardware"
 rcvar=dsbdriverd_enable
 pidfile=@PATH_PIDFILE@
 
 load_rc_config $name
 
 : ${dsbdriverd_enable:="NO"}
 
+dsbdriverd_start() {
+	if [ -n "$dsbdriverd_blacklist" ]; then
+		dsbdriverd_blacklist="-x $(echo $dsbdriverd_blacklist |
```

Comment:
I guess you meant to write `dsbdriverd_flags=` instead of `dsbdriverd_blacklist=`